src/kw_config_loader.sh

```diff
@@ -142,8 +145,25 @@ function parse_configuration()
 function load_configuration()
 {
   parse_configuration "$KW_ETC_DIR/$CONFIG_FILENAME"
-  parse_configuration "$HOME/.kw/$CONFIG_FILENAME"
-  parse_configuration "$PWD/$CONFIG_FILENAME"
+  parse_configuration "$HOME/$KW_DIR/$CONFIG_FILENAME"
+
+  # New users may not have kworkflow.config at $PWD
+  if [[ -f "$PWD/$CONFIG_FILENAME" ]]; then
+    warning 'We will stop supporting kworkflow.config soon.'
```

Comment:
Maybe "We will stop to support kworkflow.config in the kernel root directory in favor of using a .kw/ directory."